### ISSUE TITLE
Related forms do not "pop-up"

### ISSUE THREAD

**Issue by GracefulLemming** [NEU-DSG/dailp-encoding#34]
When clicking a tag on a document, known occurrences and tag name appear in the page footer; related documents do not appear (see image). This issue was first discovered in EFN4, and further snooping is ongoing. 

<img width="1440" alt="Issue_Oct 14" src="https://user-images.githubusercontent.com/22583784/96027149-78e84980-0e25-11eb-900a-ca8c54fe4c9a.png">

**Comment by loafofpiecrust** [comment]
I think I solved this with the linked commits above if you could test with the same process again @GracefulLemming?

**Comment by loafofpiecrust** [comment]
I believe that this issue is dead, once and for all, especially after #82. Open a new issue if you can reproduce it now.
